File: week08/homework/cpp/709-to-lower-case.cpp
class Solution {
public:
    string toLowerCase(string str) {
        int n = str.size();
        for (int i = 0; i < n; ++i)
        {
            if (str[i] >= 65 && str[i] <= 90)
            {
                str[i] += 32;
            }
        }
        return str;
    }
};

File: week08/homework/python/709-to-lower-case.py
# 解题1：lower() 内置函数
class Solution:
    def toLowerCase(self, s: str) -> str:
        return s.lower()

# 解题2：ASCII码 转换
class Solution:
    def toLowerCase(self, s: str) -> str:

        ansArray = []
        for i in s:
            if 65 <= ord(i) <= 90:
                ansArray.append(chr(ord(i) + 32))
            else:
                ansArray.append(i)
        return ''.join(ansArray)
